setup.py

```diff
@@ -240,8 +240,8 @@ def get_exts_for(name):
         "funcsigs>=1.0.0; python_version=='2.7'",
         "typing; python_version<'3.5'",
         "packaging>=17.1",
+        "protobuf>=3,<4.0; python_version<'3.7'",
```

Comment:
what about Python >=3.8 ?

Reply:
how does this play with `python_version<=3.6` ? not sure I've played with overlapping env constraints like this before

Reply:
yeah not sure either